cg/constants/encryption.py

```diff
@@ -1,9 +1,11 @@
 """Constants specific for encryption"""
+from enum import Enum
+from typing import List
 
-from cg.utils.enums import ListEnum, StrEnum
+from cg.utils.enums import ListEnum
 
 
-class CipherAlgorithm(StrEnum):
+class CipherAlgorithm(str, Enum):
```

Comment:
Why do we want to stop using StrEnum?

Reply:
It something custom created by Patrik Grenfeldt (in cg). Better to use core libs to do the same thing